.github/actions/amazon-linux-build-action/Dockerfile

```diff
@@ -5,8 +5,8 @@ FROM amazonlinux:latest
 RUN yum install -y tar gzip gcc
 
 # install Go
-ENV GOLANG_VERSION 1.20.2
-ENV GOLANG_VERSION_SHA256 4eaea32f59cde4dc635fbc42161031d13e1c780b87097f4b4234cfce671f1768
+ENV GOLANG_VERSION 1.21.0
```

Comment:
should we update this 1.21.1? As it's out?

Reply:
I can do that.